src/unix/linux_like/android/mod.rs

```diff
@@ -3506,6 +3506,8 @@ extern "C" {
         longopts: *const option,
         longindex: *mut ::c_int,
     ) -> ::c_int;
+
+    pub fn syncfs(fd: ::c_int) -> ::c_int;
```

Comment:
Could you add it to https://github.com/rust-lang/libc/blob/master/libc-test/semver/android.txt?

Reply:
Done. PTAL